src/Microsoft.AspNetCore.Razor/CodeGenerators/GeneratedTagHelperContext.cs

```diff
@@ -38,6 +38,8 @@ public GeneratedTagHelperContext()
             TagHelperOutputIsContentModifiedPropertyName = "IsContentModified";
             TagHelperOutputContentPropertyName = "Content";
             TagHelperOutputGetChildContentAsyncMethodName = "GetChildContentAsync";
+            TagHelperAttributeTypeName = "Microsoft.AspNetCore.Razor.TagHelpers.TagHelperAttribute";
+            HtmlStringTypeName = "Microsoft.AspNetCore.Mvc.Rendering.HtmlString";
```

Comment:
This is the first case I've seen where it would actually behoove us to just have one type.